tests/voq/test_fabric_cli_and_db.py

```diff
@@ -0,0 +1,288 @@
+from tests.common.helpers.assertions import pytest_assert
+import logging
+import random
+logger = logging.getLogger(__name__)
+
+# There are 12 asic on Supervisor now.
+# Initialize the reference data dictionary for sup.
+num_asics = 12
+
+# Set the number of links to test for each test. If all
+# links are tested these tests can take almost an hour!
+num_links_to_test = 6
+
+# This test iterates over the fabric links on a linecard
+# It isolates and unisolates each fabric link. Each time the
+# state of the link is changed the state is checked in both
+# CONFIG_DB and APPL_DB.
+# The values in CONFIG_DB are updated by the fabric CLI commands
+# and the values in APPL_DB are updated by the fabric manager
+# daemon.
+
+
+def test_fabric_cli_isolate_linecards(duthosts, enum_frontend_dut_hostname):
```

Comment:
add condtion to run this test for Voq T2 chassis only

Reply:
done